packages/react-liveness/src/components/FaceLivenessDetector/FaceLivenessDetectorCore.tsx

```diff
@@ -43,6 +43,21 @@ export default function FaceLivenessDetectorCore(
     },
   });
 
+  const [state, send] = useActor(service);
+  const isStartView = state.matches('start');
```

Comment:
`disableStartScreen` and `isStartView` read as somewhat contradicting one another imo, and since the existing declarations of `isStartView` are declared as:
```
const isStartView = state.matches('start') || state.matches('userCancel');
```

Maybe we can have a variable name spaced as something like `shouldBeginLivenessCheck` , example:
```
const shouldBeginLivenessCheck = disableStartScreen && state.matches('start');
```

Which would simplify the logic inside/deps required in the callback passed to `useLayoutEffect`